README.md

```diff
@@ -65,27 +83,27 @@ VMware vSphere Kubernetes Service (VKS, formerly known as the VMware Tanzu Kuber
 
 The [Interoperability Matrix](https://interopmatrix.broadcom.com/Interoperability?col=820,18034,18430,18431,18448,18667,18589,18562,19031,19029,19027,19025&row=2,%261794,&isHidePatch=false&isHideLegacyReleases=false) shows each VKS version below, including compatible Kubernetes releases and the vCenter Server versions containing compatible Supervisor versions. Note that some compatible Kubernetes releases may have reached End of Service; refer to the [Product Lifecycle](https://support.broadcom.com/group/ecx/productlifecycle) tool (Division: "VMware Cloud Foundation", Product Name: "vSphere Kubernetes releases") to view End of Service dates for Kubernetes releases.
 
-- Download latest version [VKS v3.4.0](https://support.broadcom.com/group/ecx/productfiles?subFamily=vSphere%20Supervisor%20Services&displayGroup=vSphere%20Kubernetes%20Service&release=3.4%2Bv1.33&os=&servicePk=&language=EN) from the [vSphere Supervisor Services section of the Broadcom Support Portal](https://support.broadcom.com/group/ecx/productdownloads?subfamily=vSphere%20Supervisor%20Services)
+- Latest version VKS v3.4.0
```

Comment:
I don't think we need to keep the `latest` text, that's obvious from the version numbering scheme and edits to this page often create typos